pkg/docker/index.html

```diff
@@ -557,7 +557,10 @@ <h4 class="modal-title" translate>Image Search</h4>
           </div>
         </div>
         <div class="modal-footer">
-          <input type="text" id="containers-search-tag" class="form-control" disabled="disabled" placeholder="Tag" />
+          <div class="ct-form containers-search-tag-form">
+            <label class='control-label' for="containers-search-tag" translate="yes">Tag</label>
+            <input type="text" id="containers-search-tag" class="form-control" disabled="disabled" placeholder="Tag" />
```

Comment:
placeholder should be translated as well.

Reply:
Ups - this need to by mustache now then in order to pass the translated string. I'd rather drop the placeholder - it does not offer any information anyway.

Reply:
you can just do `translate="placeholder"`, or it does not work in this case?

Reply:
I will leave this as is - as it's hard to get tests green and get the fixes for this and the function name in a followup

Reply:
As this needs rebasing anyway, you may just as well do it now?